docs/collector_module.md

```diff
@@ -128,10 +134,14 @@ security-server:
     tls-client-key: /path/to/client.key # path to client's private key
     tls-server-certificate: /path/to/server.crt # path to server's certificate
 ```
-Notes:
-  Client's certificate has to be sent to security server administrator.
-  Server certificate has to be sent by server's administrator and save in client's location.
-  `tls-server-certificate` can be set to `False` to disable server certificate verification.
+
+> [!IMPORTANT]
+> - Client's certificate has to be sent to security server administrator.
+> - Server certificate has to be sent by server's administrator and save in client's location.
+
+> [!Note]  
+> **For non-production environments only**:
+> To disable server certificate verification, set `tls-server-certificate` to `False`.
```

Comment:
We could soften this a bit. The component is usually used internally so using `HTTP` or using `HTTPS NO AUTH` isn't such a big issue.

I would change the wording a bit so that it says that disabling verification is not suggested in production, but lets not go as far as to say it shouldn't be disabled in production at all.

Reply:
I reverted the wordings back to the original ones and kept it as a list item in PR #112